### ISSUE TITLE
Is it possiable that the prediction result would be nested for NER task?

### ISSUE THREAD

**Issue by MingHong1019** [markus-eberts/spert#42]
to be specifically,  for example, let the sentence "The apple corporation" as input, is there any possible that the span ["The apple corporation"] is predicted as the label 'ORG' however, the span  ["The apple"] is predicted as label the 'Other'?  If so,  what is the final prediction of input sentence "The apple corporation"?  

by the way, in the training progress, after random sample, the max neg-span number is 100. But it is also seems much larger than the number of label span. For example, the sentence is made by 27 word, and the number of entities are 5, hence the label_entity_ span will be 5 whereas the neg_entity_span are 100. Will it be harmful for training the model?

**Comment by markus-eberts** [comment]
1. Since we are using a span based approach, SpERT indeed is able to detect overlapping entity mentions. So in your example, SpERT would return two mentions, "The Apple" labeled with "Other" and "The Apple cooperation" labeled with "Org". Of course this requires your training data to contain such overlapping samples.

2. I think we had a similar ratio in the datasets used in our paper and it worked out fine. As always, a (automated or manual) hyperparameter search may be useful when working with a new dataset. You can change the negative entity count in the config file.

**Comment by markus-eberts** [comment]
I assume this answered your question, please write a comment otherwise.
